CMakeLists.txt

```diff
@@ -7,6 +7,12 @@ find_package(Boost REQUIRED)
 find_package(mongocxx REQUIRED)
 find_package(Python REQUIRED Development)
 
+include(GNUInstallDirs) # Required for ${CMAKE_INSTALL_LIBDIR}

```

Comment:
The `GNUInstallDirs` module should be used for using common abstracted install locations (which match common defaults on GNU systems like Linux): https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html